### ISSUE TITLE
Display aggregate PSI stats on main dashboard 

### ISSUE THREAD

**Issue by marrrmarrr** [google/site-kit-wp#1439]
## Feature Description

Right now we display PSI stats for only the home page on the main Site Kit dashboard. Let's change this to give an overall view of how the entire site is doing. 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* On the main Site Kit dashboard, replace PSI stats for the home page with PSI stats for the entire domain.  
* Replace "How fast your home page is." with "How fast your site is."

## Implementation Brief

* 

## QA Brief

* 

## Changelog entry

*

**Comment by adamsilverstein** [comment]
@marrrmarrr Digging into this a bit further and testing I discovered an API shortcoming I wasn't aware of previously:

When testing with my personal site passing `origin:{site_url}` I received **no data**. When testing with a popular site url (google.com) I did get data, however this was only the CrUX data provided by the API - (first contentful paint and first input delay). This makes some sense that the lighthouse data is not included - the lighthouse data is generated per url on the fly, while. Also, this data may not be available for all sites.

In short, my reading of the documentation on `origin:`  was overly optimistic: we can't use that for whole site **lighthouse** reports. 

* We could possibly display these additional metrics as site wide metrics, they do provide an overall score like "Average" "Good" or "Bad"
* We could consider providing lighthouse scores for more than the homepage on the dashboard, eg show the homepage plus the url of the most popular page from analytics.
